hedyorg/hedy
# [BUG FIX] Fixes empty modal alert shown on save/share and copy to clipboard action

**Description**
In this PR we fix an issue where the modal alert() would be shown empty once the user saves/shares their programs and presses the "copy to clipboard" button. We add an additional response to make sure that the alert have some useful text as well as hiding the copy modal once the button is pressed.

**Fixes**
This PR fixes #3493

**How to test**
Make sure you are logged in and save/share a program. Click the "copy to clipboard" option and verify that the modal hides and a nice modal alert() is shown with the feedback that the copying has been successful.
